Source/MultipartFormData.swift

```diff
@@ -24,11 +24,13 @@
 
 import Foundation
 
+#if canImport(Darwin)
```

Comment:
This Darwin checks seems incorrect. Probably better to redo the OS check as a `canImport` for `MobileCoreServices` or `CoreServices.